integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -393,8 +402,12 @@ public String getFSName() {
   @Override
   public int mkdir(String path, @mode_t long mode) {
     final AlluxioURI turi = mPathResolverCache.getUnchecked(path);
+    final String name = turi.getName();
     LOG.trace("mkdir({}) [Alluxio: {}]", path, turi);
     try {
+      if (name.length() > MAX_NAME_LENGTH) {
+        throw new NameTooLongException(name + " beyond " + MAX_NAME_LENGTH + " characters");
```

Comment:
same here, replace throw and catch with `return -ErrorCodes.ENAMETOOLONG()` directly? We should actually move this outside `try` block as the sanity check

Reply:
modified.